uui/components/datePickers/DatePicker.tsx

```diff
@@ -10,7 +10,12 @@ import { DropdownContainer } from '../overlays';
 const defaultMode = EditMode.FORM;
 
 /** Represents the properties of the DatePicker component. */
-export interface DatePickerProps extends DatePickerCoreProps, SizeMod, IHasEditMode {}
+export interface DatePickerProps extends DatePickerCoreProps, SizeMod, IHasEditMode {
+    /**
+    * Defines id for inner input
```

Comment:
Let's change to - 
    /** HTML ID attribute for the toggler input */

Reply:
fixed